TramsDataApi/UseCases/AddAcademyConversionProjectNote.cs

```diff
@@ -0,0 +1,39 @@
+using System;
+using System.Linq;
+using TramsDataApi.DatabaseModels;
+using TramsDataApi.Factories;
+using TramsDataApi.RequestModels.AcademyConversionProject;
+using TramsDataApi.ResponseModels.AcademyConversionProject;
+
+namespace TramsDataApi.UseCases
+{
+    public class AddAcademyConversionProjectNote : IAddAcademyConversionProjectNote
+    {
+        private readonly TramsDbContext _tramsDbContext;
+
+        public AddAcademyConversionProjectNote(TramsDbContext tramsDbContext)
+        {
+            _tramsDbContext = tramsDbContext;
+        }
+
+        public AcademyConversionProjectNoteResponse Execute(int academyConversionProjectId, AddAcademyConversionProjectNoteRequest request)
+        {
+            if(_tramsDbContext.AcademyConversionProjects.SingleOrDefault(p => p.Id == academyConversionProjectId) == null)
```

Comment:
Use `if (_tramsDbContext.AcademyConversionProjects.Any(p => p.Id == academyConversionProjectId))` that way you just check if it exists rather than retrieving